python/pyspark/pandas/generic.py

```diff
@@ -2619,6 +2619,56 @@ def expanding(self: FrameLike, min_periods: int = 1) -> "Expanding[FrameLike]":
 
         return Expanding(self, min_periods=min_periods)
 
+    # TODO: 'adjust', 'ignore_na', 'axis', 'method' parameter should be implemented.
+    def ewm(
```

Comment:
Would have to list it in https://github.com/apache/spark/blob/master/python/docs/source/reference/pyspark.pandas/window.rst so it's documented.